docs/architecture/architecture.md

```diff
@@ -373,7 +373,7 @@ signals, `cc-runtime` will kill the `cc-shim`.  All other signals are ignored.
 #### [`delete`](https://github.com/clearcontainers/runtime/blob/master/delete.go)
 
 `delete` is about deleting all resources held by a stopped/killed container.
-Running containers can not be deleted unless the OCI runtime is explictly being
+Running containers can not be deleted unless the OCI runtime is explicitly being
```

Comment:
Running containers cannot be deleted...

Reply:
Thanks @klynnrif - branch updated with fixes for three occurrences of that issue ;)